src/stdlib/redact.rs

```diff
@@ -53,6 +58,30 @@ impl Function for Redact {
                 source: r#"redact({ "name": "John Doe", "ssn": "123-12-1234"}, filters: ["us_social_security_number"])"#,
                 result: Ok(r#"{ "name": "John Doe", "ssn": "[REDACTED]" }"#),
             },
+            Example {
+                title: "text redactor",
+                source: r#"redact("my id is 123456", filters: [r'\d+'], redactor: {"type": "text", "replacement": "***"})"#,
+                result: Ok(r#"my id is ***"#),
+            },
+            Example {
+                title: "sha2",
+                source: r#"redact("my id is 123456", filters: [r'\d+'], redactor: "sha2")"#,
+                result: Ok(r#"my id is GEtTedW1p6tC094dDKH+3B8P+xSnZz69AmpjaXRd63I="#),
```

Comment:
🥜 Would it better if we wrapped the result e.g. 
```
Ok(r#"my id is `GEtTedW1p6tC094dDKH+3B8P+xSnZz69AmpjaXRd63I=`"#),
```

Reply:
maybe? I don't have a preference either way.

Reply:
In this case, let's leave it as is